includes/orders/functions/actions.php

```diff
@@ -310,17 +310,22 @@ function edd_add_manual_order( $args = array() ) {
 				if ( isset( $download['adjustments'] ) ) {
 					$order_item_adjustments = array_reverse( $download['adjustments'] );
 
-					foreach ( $order_item_adjustments as $order_item_adjustment ) {
+					foreach ( $order_item_adjustments as $index => $order_item_adjustment ) {
 
 						// Discounts are not tracked at the Order Item level.
 						if ( 'discount' === $order_item_adjustment['type'] ) {
 							continue;
 						}
 
+						$type_key = ! empty( $order_item_adjustment['description'] )
```

Comment:
It is probably not necessary, but should the description as fallback also be cast to lowercase?